docs/vscode.md

```diff
@@ -559,8 +606,8 @@ The following settings are available:
 `nextflow.java.home`
 : Specifies the folder path to the JDK. Use this setting if the extension cannot find Java automatically.
 
-`nextflow.suppressFutureWarnings`
-: Hide warnings for future changes, deprecations, and removals.
+`nextflow.paranoidWarnings`
+: Enable additional warnings for things like future deprecations, discouraged patterns, and so on.
```

Comment:
```suggestion
: Enable additional warnings for future deprecations, discouraged patterns, and other upcoming changes.
```
Are these additional warnings? Or are these warnings only shown if this is enabled?

Reply:
They are both